source/geod24/LocalRest.d

```diff
@@ -110,24 +110,11 @@ import std.traits : fullyQualifiedName, Parameters, ReturnType;
 import core.thread;
 import core.time;
 
-/// Request / Response ID
-private struct ID
-{
-    /// A node may restart, in which case it will spawn a new request scheduler
-    size_t sched_id;
```

Comment:
This is suspicious. I don't see why the restart logic should be affected here.

Reply:
Command ids persist through restarts. So sched_id is not needed.